src/briefcase/integrations/linuxdeploy.py

```diff
@@ -48,6 +49,21 @@ def download_url(self) -> str:
     def file_path(self) -> Path:
         """The folder on the local filesystem that contains the file_name."""
 
+    @classmethod
+    def arch(cls, host_os: str, host_arch: str) -> str:
+        # always use the x86-64 arch on macOS since Docker
+        # containers are always run in an x86-64 VM
+        arch = host_arch if host_os != "Darwin" else "x86_64"
+        try:
+            return {
+                "x86_64": "x86_64",
+                "i686": "i386",
+            }[arch]
```

Comment:
I believe the source of this arch mapping is from AppImageTool....however, this [commit](https://github.com/AppImage/appimagetool/commit/7a10b890312d9f0963a88eb39bc3897afa2e36b8) makes me think it might start using `i686` "soon"....I don't really understand how these things package and reference each other....

Reply:
Yeah - given Appimage/LinuxDeploy has a very loose understanding of versioning, we can pretty much only go with what works now, and adapt when/if we discover a problem in the future.